tasks/tasks/gadm_processors.py

```diff
@@ -17,83 +17,27 @@ def resolution_alternatives(context, filename=None, **kwargs):
 
     admin_level = kwargs.get("admin_level", "country")
 
+    has_primary_geo = False
+
     geo_column = ""
     for geo in context.get("annotations").get("annotations").get("geo"):
         if geo.get("primary_geo", False):
+            has_primary_geo = True
             geo_type = admin_level_to_geo_type(admin_level=admin_level)
             if geo.get("geo_type") == geo_type:
                 geo_column = geo.get("name")
                 break
             geo_column = geo.get("name")
 
+    if not has_primary_geo:
+        return {
+            "field": None,
+            "fuzzy_match": []
+        }
+
     # Run job
     gadm_response = elwood.get_gadm_matches(original_dataframe, geo_column, admin_level)
 
-    # Return hardcoded response in job
-    # provide gadm alternatives
-    mock_gadm_alternatives = {
```

Comment:
Removed this old mock data, which was useful to illustrate the response body, but will easily get stale.